docs/en/infraops/installation.asciidoc

```diff
@@ -91,6 +92,52 @@ module]
 * {filebeat-ref}/filebeat-input-docker.html[{filebeat} `docker` input]
 * {filebeat-ref}/add-kubernetes-metadata.html[{filebeat} `add_kubernetes_metadata` processor]
 
+[float]
+==== Which fields are used for the metrics on the map?
+
+
```

Comment:
@dedemorton I feel like there should be some text here but I'm not sure what it should be. Maybe something about how using Beat Shippers listed above automatically sends these fields. Along with these fields, documents must also contain the identifying fields for each node type (host, container, pod).

Reply:
Yes. In fact, you need to have text here, or the doc build fails.